guides/common/modules/proc_configuring-an-external-dhcp-server.adoc

```diff
@@ -34,10 +34,10 @@ As a result, a key pair that consists of two files is created in the current dir
 +
 [options="nowrap"]
 ----
-# cat Komapi_key.+*.private |grep ^Key|cut -d ' ' -f2
+# grep ^Key Komapi_key.+*.private  | cut -d ' ' -f 2
```

Comment:
Why read the whole file when you can directly grep. This command looks better to me.
Just a suggestion.